contracts/lockdrop/README.md

```diff
@@ -37,13 +37,14 @@ Note - Users can open muliple lockup positions with different lockup duration fo
 
 ### Query Messages
 
-| Message                | Description                                                                                                      |
-| ---------------------- | ---------------------------------------------------------------------------------------------------------------- |
-| `QueryMsg::Config`     | Returns the config info                                                                                          |
-| `QueryMsg::State`      | Returns the contract's global state                                                                              |
-| `QueryMsg::Pool`       | Returns info regarding a certain supported LP token pool                                                         |
-| `QueryMsg::UserInfo`   | Returns info regarding a user (total ASTRO rewards, list of lockup positions)                                    |
-| `QueryMsg::LockUpInfo` | Returns info regarding a particular lockup position with a given duration and identifer for the LP tokens locked |
+| Message                         | Description                                                                                                      |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------|
+| `QueryMsg::Config`              | Returns the config info                                                                                          |
+| `QueryMsg::State`               | Returns the contract's global state                                                                              |
+| `QueryMsg::Pool`                | Returns info regarding a certain supported LP token pool                                                         |
+| `QueryMsg::UserInfo`            | Returns info regarding a user (total ASTRO rewards, list of lockup positions)                                    |
+| `QueryMsg::LockUpInfo`          | Returns info regarding a particular lockup position with a given duration and identifer for the LP tokens locked |
+| `QueryMsg::PendingAssetReward`  | Returns the amount of pending asset rewards for the specified recipient                                          |
```

Comment:
It returns pending rewards for a specific lockup position not only for a specific user.

Reply:
Ok, changed